pkg/search/index.go

```diff
@@ -14,12 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/*Package search provides client-side repository searching.
```

Comment:
why did you remove the package description?

Reply:
Now it lives on the doc.go

When there is a bit of text around in the packages, I prefer to extract it to a doc.go file, seems nicer and all the description adn explanation are together.